patches/react-native-modal+13.0.1.patch

```diff
@@ -58,7 +265,40 @@ index 80f4e75..5a58eae 100644
      }
      componentDidUpdate(prevProps) {
          // If the animations have been changed then rebuild them to make sure we're
-@@ -525,7 +543,7 @@ export class ReactNativeModal extends React.Component {
+@@ -475,9 +510,14 @@ export class ReactNativeModal extends React.Component {
+             this.animationOut = animationOut;
+         }
+         // If backdrop opacity has been changed then make sure to update it
+-        if (this.props.backdropOpacity !== prevProps.backdropOpacity &&
+-            this.backdropRef) {
+-            this.backdropRef.transitionTo({ opacity: this.props.backdropOpacity }, this.props.backdropTransitionInTiming);
++        if (this.props.backdropOpacity !== prevProps.backdropOpacity) {
++            this.setState((prevState) => ({
++                backdrop: {
++                    prevOpacity: prevState.backdrop.opacity,
++                    opacity: this.props.backdropOpacity,
++                    duration: this.props.backdropTransitionInTiming,
```

Comment:
I'm not sure why you are storing the opacity value which comes from props into state. Why can't you just use it directly from props?

Reply:
That's fine. But instead of saving it in the state, can't you just use the `usePrevious` hook? It would store the previous value in a ref which will prevent additional renders.

Reply:
Ah, makes sense. I did not consider that this was a class component.